### ISSUE TITLE
No questions shown - following tutorial

### ISSUE THREAD

**Issue by RobTrehy** [robinheinze/ignite-trivia#6]
I am following Part 2 of the tutorial. I've reached the part just before the `Making It Interactive` heading.

The app is not displaying any questions. I've used console.log to trace where the code gets to.

Nothing seems to be returned on the line `const result: GetQuestionsResult = yield self.environment.api.getQuestions()` from question-store.ts
If I console.log `result`, I don't even see any console log

However, in api.ts. I can log

```
const convertedQuestions: QuestionSnapshot[] = rawQuestions.map(convertQuestion)
console.log(convertedQuestions)
```
and I see the questions in the console.
It seems to not get any further.

I've compared code with that of the repo, but can't see anything missing.

I'm new to react-native and mobile app development, but familiar with React. I'm also new to TypeScript.
Any pointers where I could be looking would be appreciated :)

Edit to add:
My environment is Windows 10 21H1, using Android Studio - Android 10 emulator
Running the package using React-Native

**Comment by phhoef** [comment]
Exact same problem here.
Running on macOS and iOS Simulator. App has been setup with expo.

When having a look in the template generated by the cli, the user character-store is not using yield, but await.
When reading the MobX docs, it's pointed out, that `flow` should be used.

@RobTrehy did you find any further information?
@robinheinze Can you please give us some guidance?

EDIT:
The template app, generated by the ignite-cli, does not use flow/yield, but async/await.
But even when I change the action to async/await, it is not working. It seems, that the code after await cannot be reached, too.

**Comment by phhoef** [comment]
It seems, that `apisauce` is the problem.
When the api is returning a promise with a delay (`setTimeout`), it is in fact working.
That said, I can log the result of the request.
I don't really understand what's going on.

**Comment by kateinkim** [comment]
Hi @RobTrehy and @phhoef!
Due to the current issue with [Ignite](https://github.com/infinitered/ignite/issues/1769), we have been recommending people add a `setTimeout` with at least 1ms to call the data. (You can see the comment left on the new [updated code](https://github.com/robinheinze/ignite-trivia/blob/83c1ef17d40a0f6c293393244a15b4d5d864b0ce/app/screens/question/question-screen.tsx#L60)) However, as you can see on the Ignite issue thread, the problem seems like it's coming from `ToggleStorybook`. You can either remove it from `app.tsx` or you can leave it as is since the bug is only in debug configuration. We are trying to figure out why `ToggleStorybook` is causing this issue. Thanks!

**Comment by phhoef** [comment]
Hi @kateinkim,

thanks for your answer.
I was able to solve my problem a few minutes ago. I re-used the screen created with the template project.
It looked like this:
```<Screen style={ROOT} preset="scroll">```
I added a `FlatList` to show several items received from the api call. The problem obviously was, that there were two scroll views nested. I just removed the preset and the problem was gone.

I am new to react native and ignite, so I do not completely understand what happened and why this was a problem.
But for now, the problem is solved :-)

That said, really appreciate your input and I'll adapt my code to match your recommendations.
